Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections;
+using System.Collections.Generic;
+using System;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+
+    public int speed;
```

Comment:
```suggestion
    public float Speed;
```
Due to code style requirements for **fields**.
Consider to use **float** type instead **int**.